src/index.js

```diff
@@ -20,12 +20,30 @@ const client = new ApolloClient({
   cache: new InMemoryCache(),
 });
 
-ReactDOM.render(
-  <ApolloProvider client={client}>
-    <App />
-  </ApolloProvider>,
-  document.getElementById('root')
-);
+function renderApp(config) {
+  ReactDOM.render(
+    <ApolloProvider client={client}>
```

Comment:
If you want to test the logic at all, you'd probably want to separate the JSX syntax from the call to `ReactDOM.render()`, and move it to a function that returns the virtual DOM element (in other words, a React component).

Reply:
Please have a look into changes made [here](https://github.com/opentargets/genetics-app/pull/125/commits/cfc5e04070c16491796895612cee2082b4e9ad39)